docs/manage/partitions/flex/create-edit-partitions-flex.md

```diff
@@ -66,7 +66,8 @@ You can make some changes to an existing partition:  
   By default, Sumo Logic internal partitions like `sumologic_audit_events`, `sumologic_volume`, and so on, have the same retention period as the Default Continuous Index. You can change the retention period for any of these internal partitions as desired.
   :::
 * You can change the data forwarding configuration.
-* You cannot change the name of partition, routing expression, or reuse a partition name.  
+* You cannot change the name of partition, routing expression, or reuse a partition name.
+* You cannot edit the audit index partition to include in the default scope. 
```

Comment:
```suggestion
* You cannot change the name of a partition, the routing expression, or reuse a partition name.
* You cannot edit the audit index partition to include it in the default scope. 
```

Reply:
@kimsauce, I'm confused by this statement: "You cannot change the name of partition, routing expression, or reuse a partition name." A little further down, the dialog to edit a partition seems to show that you can change the name of a partition and its routing expression.

Reply:
@jpipkin1 we cannot edit/change the partition name and routing expression once it is created. Even if you check the next steps on **How to edit a partition** you can just see we only allow editing of retention period and modifying the configuration of data forwarding.

Reply:
Thanks for the explanation, @JV0812.